modules/puppet_proxy/puppet_config.rb

```diff
@@ -1,4 +1,4 @@
-require 'augeas'
+require 'augeas' if RUBY_PLATFORM =~ /ruby/
```

Comment:
This is a bug I introduced in https://github.com/theforeman/smart-proxy/pull/317. Looking into this, but assume that if a module is disabled its dependencies will not be loaded.